addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -154,6 +157,11 @@ FormRenderer.include({
             await this._mountChatterContainerComponent();
         }
     },
+    _reloadRecord() {
```

Comment:
guideline: methods defined in alphabetical order, also I would rename this to better match what it does (prompt reload?)